src/connection-monitor-preview/azext_connection_monitor_preview/tests/latest/test_connection_monitor_preview.py

```diff
@@ -14,9 +14,6 @@ class ConnectionMonitorPreviewScenarioTest(ScenarioTest):
     def _prepare_vm(self, resource_group, vm):
         vm_create_cmd_tpl = 'vm create -g {rg} --name {vm} ' \
                             '--image UbuntuLTS ' \
-                            '--authentication-type password ' \
-                            '--admin-username deploy ' \
-                            '--admin-password PassPass10!) ' \
```

Comment:
Why these argument removed?

Reply:
Because there are some vm rules in my subscription which forbid us to use this authentication-type